hive-metastore/src/test/java/org/apache/iceberg/hive/TestClientPool.java

```diff
@@ -19,6 +19,7 @@
 
 package org.apache.iceberg.hive;
 
+import org.apache.iceberg.ClientPool;
```

Comment:
This test should also be moved if the original class was moved.

Reply:
my bad, thank you for pointing it.